### DISCUSSION TITLE
After the upgrade of 1.15, the farmland that “Plots Passed Filter” and “Proofs Found” can no longer be seen in the log.

### DISCUSSION THREAD

**Discussion by coffeebreak2046** [Chia-Network/chia-blockchain#4762] [Support]
In the past, you can see through the log how many plots Passed Filter & “Proofs Found” and network response time, but after upgrading 1.15, it is found that this information no longer show in the log.
Information like this:
2021-05-05T12:10:03.465 harvester chia.harvester.harvester: INFO 1 plots were eligible for farming 3d310f4c3a... Found 1 proofs. Time: 0.10930 s. Total 96 plots

**Comment by googgiig9908** [comment]
me too, we need help with that.... Follow up

**Comment by lvcivs** [comment]
Maybe the log level got reset during the upgrade? You need to open your `config.yaml` file and make sure the log level is set to INFO.

**Reply by coffeebreak2046** [reply]
Thanks for the advice. I have confirmed that config.yaml is set to INFO. Maybe this is not the reason.

**Comment by googgiig9908** [comment]
I found the way to fix it
You need to open your .chia ---> mainnet ---> config and open   ---->config.yaml
![Screenshot (5)_LI](https://user-images.githubusercontent.com/82992332/118108301-087cc780-b40a-11eb-8c63-9673b0a7bd8c.jpg)
at full _node _pear
host: loalhost

change host to localhost on both of them

**Reply by coffeebreak2046** [reply]
Thanks,I check the config.yaml file, It's already been set up like below，but still no plots Passed Filter information.
farmer:
  full_node_peer:
    host: localhost
    port: 8444
  harvester_peer:
    host: localhost
    port: 8448
  logging: &id001
    log_filename: log/debug.log
    log_level: INFO
    log_stdout: false

**Reply by googgiig9908** [reply]
have you restart the computer yet?

**Reply by coffeebreak2046** [reply]
No,because the host already set up to localhost, I didn`t change it.

**Reply by dslrnation** [reply]
I have the same problem and I searched everywhere and also happened after installing 1.1.5 nothing is showing, I changed it to info, I restarted chia, PC and restarted my brain still nothing :p 

![image](https://user-images.githubusercontent.com/79029824/118141771-bc189400-b3d7-11eb-9f24-bf7ede655b44.jpeg)

**Reply by dslrnation** [reply]
Worked for me too finally, I changed it localhost and restarted pc, thanks alot

**Reply by rcotgrov** [reply]
am having a similar problem. Enabled INFO but the localhost was set correctly. Restarted the daemons but have not yet rebooted PC. Will try this. See what happens

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
